evalml/tests/pipeline_tests/test_pipeline_utils.py

```diff
@@ -529,21 +455,21 @@ def test_stacked_estimator_in_pipeline(problem_type, X_y_binary, X_y_multi, X_y_
         X, y = X_y_binary
         base_pipeline_class = BinaryClassificationPipeline
         stacking_component_name = StackedEnsembleClassifier.name
-        input_pipelines = [make_pipeline_from_components([classifier], problem_type) for classifier in stackable_classifiers]
+        input_pipelines = [base_pipeline_class([classifier]) for classifier in stackable_classifiers]
         comparison_pipeline = logistic_regression_binary_pipeline_class(parameters={"Logistic Regression Classifier": {"n_jobs": 1}})
         objective = 'Log Loss Binary'
     elif problem_type == ProblemTypes.MULTICLASS:
         X, y = X_y_multi
         base_pipeline_class = MulticlassClassificationPipeline
         stacking_component_name = StackedEnsembleClassifier.name
-        input_pipelines = [make_pipeline_from_components([classifier], problem_type) for classifier in stackable_classifiers]
+        input_pipelines = [base_pipeline_class([classifier]) for classifier in stackable_classifiers]
         comparison_pipeline = logistic_regression_multiclass_pipeline_class(parameters={"Logistic Regression Classifier": {"n_jobs": 1}})
         objective = 'Log Loss Multiclass'
     elif problem_type == ProblemTypes.REGRESSION:
         X, y = X_y_regression
         base_pipeline_class = RegressionPipeline
         stacking_component_name = StackedEnsembleRegressor.name
-        input_pipelines = [make_pipeline_from_components([regressor], problem_type) for regressor in stackable_regressors]
+        input_pipelines = [base_pipeline_class([regressor]) for regressor in stackable_regressors]
```

Comment:
nit-pick: I don't think there's a need for `base_pipeline_class` anymore.

Reply:
Hmmm, I think we still need it for the line after (depending on which problem type, we create a specific pipeline class) but I'm okay with updating this line to just use RegressionPipeline so it's more clear! 😁